subsys/bluetooth/mesh/shell/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Bluetooth mesh shell configuration options
```

Comment:
Is it worth creating a separate configuration file for one option?
What is the benefit of this idea?

Reply:
For this option isolated I could agree to put it in the common Kconfig, but I highly suspect more shell options will be added in the future. In that case I would prefer to keep a separate file for it. I don't like the idea of dumping everything in a single file, since it IMO creates some noise for people trying to understand Mesh configuration. Shell is not a core feature of the mesh spec, so it might aswell keep it's configuration options in a separate file.

Reply:
Which options are you going to move here?

Reply:
Is it going to be separate PR ?

Reply:
There are not currently any other option that needs to be moved. But if I start by adding one extra option to the main Kconfig now I suspect it will make a precedence where more options will be added there later.

Reply:
IMHO this is a bit tricky. I'll not block PR due to this but I'd prefer changes for purpose.

Reply:
I start thinking that we need a separate Kconfig. Because if we add more models to shell, this will clutter the main Kconfig. I also start thinking that we (will) need to add Kconfig options to control which mesh shell commands needs to be enabled, like we do in nRF Connect SDK https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/mesh/shell/Kconfig. I think users should be able to choose which options they actually need.